packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -81,7 +82,7 @@ interface DatabaseClusterBaseProps {
   /**
    * Indicates whether the DB cluster should have deletion protection enabled.
    *
-   * @default false
+   * @default - true if ``removalPolicy`` is RETAIN
```

Comment:
Maybe to clarify this completely:

```suggestion
   * @default - true if ``removalPolicy`` is RETAIN, false otherwise
```